### ISSUE TITLE
feature request: Templates for Mise Tasks

### ISSUE THREAD

**Issue by theutz** [stevearc/overseer.nvim#387]
### Did you check existing requests?

- [X] I have searched the existing issues

### Describe the feature

[Mise](https://mise.jdx.dev) provides a JSON api for dumping it's task list that could probably be used similarly to Just's setup. It'd be awesome if those worked like the other built-in tools!

### What is the significance of this feature?

nice to have

**Comment by disrupted** [comment]
is there something left to do here after #414 ?
